CHANGELOG.md

```diff
@@ -42,6 +42,10 @@ Ref: https://keepachangelog.com/en/1.0.0/
 
 * [597](https://github.com/provenance-io/cosmos-sdk/pull/597) Provenance: Create the ReadPageRequestWithPageKeyDecoded helper from v0.50.x to v0.53.x (original PR #597).
 
+### Improvements
+
+* (x/bank) [616](https://github.com/provenance-io/cosmos-sdk/pull/616) Improve GetAllBalances and GetAccountsBalances optimization from v0.50.x to v0.53.x (original PR #616).
```

Comment:
Please update the link in this to reference this PR instead of the original. But keep the "(original PR #616)" part.

Could you also please fix the `ReadPageRequestWithPageKeyDecoded` entry's link to point to the more recent PR for it (I think it was 621).